internal/pkg/deploy/cloudformation/stack/lb_web_svc.go

```diff
@@ -189,8 +189,11 @@ func (s *LoadBalancedWebService) Template() (string, error) {
 		}
 	}
 
-	hostedZoneToAliases := convertHostedZone(s.manifest.RoutingRule.RoutingRuleConfiguration)
-	if len(hostedZoneToAliases) != 0 && !s.certImported {
+	aliasesFor, err := convertHostedZone(s.manifest.RoutingRule.RoutingRuleConfiguration)
+	if err != nil {
+		return "", err
+	}
+	if len(aliasesFor) != 0 && !s.certImported {
```

Comment:
Likely I'm missing something: does backend service need this validation?

Reply:
Not really because for backend service we don't add A record by lambda